archai/discrete_search/api/search_results.py

```diff
@@ -197,6 +197,7 @@ def plot_2d_pareto_evolution(
         status_df = self.get_search_state_df().copy()
 
         fig, ax = plt.subplots(figsize=figsize)
+        fig.patch.set_facecolor('white')
```

Comment:
But does this work on a vscode with light color theme?

Reply:
Yes, this line is only changing the background of the region where the axis labels are. By default, this background is transparent, hence the problem you pointed out with dark themes and the black text of the axis labels.

![image](https://user-images.githubusercontent.com/13797878/219068587-007109b4-a674-4330-89aa-e801898282a4.png)

Reply:
Ah, I see thanks.